Yoast/developer
# Use valid url in organization example

## Summary

There's a schema example with a url referencing a 404 page. This updates that url to point somewhere else.

## Test instructions
This PR can be acceptance tested by following these steps:
1. Visit https://developer.yoast.com/features/schema/pieces/organization/#api
1. Check the example in `WordPress API: Change Organization Schema output` has a `founder` pointing to an existing url.


## Quality assurance
